docs/fix-links/filter.lua

```diff
@@ -0,0 +1,40 @@
+local len = pandoc.text.len
+local sub = pandoc.text.sub
+
+-- True if str starts with prefix
+local function hasPrefix(prefix, str)
+	local pfxLen = len(prefix)
+	local strLen = len(str)
+	if pfxLen == strLen then
+		return prefix == str
+	end
+	if pfxLen < strLen then
+		return prefix == sub(str, 1, pfxLen)
+	end
+	return false
+end
+
+function Link(link)
+	local target = link.target
+	-- Check for relative links
+	-- TODO: handle ../
+	if hasPrefix("./", target) then
+		link.target = githubUrl .. sub(target, 3)
+		return link
+	end
+	if not hasPrefix("https://", target) then
+		link.target = githubUrl .. target
+		return link
+	end
```

Comment:
This doesn't correctly handle `#` anchor targets on the same page.

We need a case that checks `hasPrefix("#")`

Reply:
Wasn't a very effective review on my part

Reply:
Don't be silly. These kinda things slip through easily and it's no worse than it was before